src/api/endpoints/activity_types.py

```diff
@@ -1,13 +1,13 @@
 """Module for activity types operations."""
 
 from flask import request
-from flask_restplus import Resource
```

Comment:
Didn't know you'd moved from restplus. Any reason?

Reply:
We had decided on this a while back, also we are using apiary for our documentation.